### DISCUSSION TITLE
How to detect and convert GET query string values that are arrays into strings for hidden fields?

### DISCUSSION THREAD

**Discussion by jrrdnx** [solspace/craft-freeform#629] [Feature Requests]
I have a custom form-based workflow built outside of Freeform, that when the user completes they have the option to fill out a contact form that _is_ built in Freeform. This contact form is built to automatically include values from the GET query string in hidden inputs so that their selections are known.

Unfortunately, one of those workflow steps includes a checkbox group so the query string now potentially includes multiple `input[]` values which causes an "Array to string conversion" error in Freeform when it attempts to set the hidden field's value.

I'm curious if this is currently possible with this particular setup, as the Freeform form is rendered automatically so I don't have a custom template available to convert the array and set the field's value manually. Or if this could be some sort of feature request to include an option on fields to convert arrays automatically?

Open to ideas, thanks!

**Comment by kjmartens** [comment]
Thanks for this feedback @jrrdnx. I've added this to our feature requests list. 🙂 

In the meantime, I suppose the only way to get this working would be to have a checkbox group or multi-select field type in Freeform collect those values (and if you want it hidden, use CSS to do that).

**Comment by jrrdnx** [comment]
I've added this as a checkbox group and hidden it with CSS for now to achieve the intended functionality. Thanks for adding this to the feature requests list!
